src/applications/disability-benefits/all-claims/pages/uploadPersonalPtsdDocuments.jsx

```diff
@@ -2,24 +2,24 @@ import React from 'react';
 
 import { UploadDescription } from '../content/fileUploadDescriptions';
 import { ptsd781aNameTitle } from '../content/ptsdClassification';
-import { ancillaryFormUploadUi } from '../utils';
-import fullSchema from 'vets-json-schema/dist/21-526EZ-ALLCLAIMS-schema.json';
+import { ancillaryFormUploadUi, setDefaultAttachmentId } from '../utils';
 
-const { attachments } = fullSchema.properties;
-const secondarySupportStatement = 'L229';
+const PTSD_781A_ATTACHMENT_ID = 'L229';
 
 export const uiSchema = {
   'ui:title': ptsd781aNameTitle,
   'ui:description': <UploadDescription uploadTitle="Upload VA Form 21-0781a" />,
-  ptsd781a: ancillaryFormUploadUi('', 'PTSD 781a form', {
-    attachmentId: secondarySupportStatement,
+  form781aUpload: ancillaryFormUploadUi('', 'PTSD 781a form', {
+    attachmentId: PTSD_781A_ATTACHMENT_ID,
```

Comment:
Have to pass the `attachmentId` both here and in the schema as `default` so it not only has the value but displays the correct docType title.